user_tools/src/spark_rapids_pytools/__init__.py

```diff
@@ -14,7 +14,10 @@
 
 """init file of the spark_rapids_pytools package."""
 
-from spark_rapids_pytools.build import get_version
+from spark_rapids_pytools.build import get_version, get_runtime_buildver
 
 VERSION = '24.08.2'
+# defines the default runtime build version for the user tools environment
+RUNTIME_BUILDVER = '350'
```

Comment:
if this is the spark version can we rename this as well to have spark version in the name of it, same with the other functions

Reply:
Thanks @tgravescs 
Renamed all `RUNTIME_BUILDVER`